saleor/graphql/product/tests/mutations/test_product_create.py

```diff
@@ -2156,61 +2156,6 @@ def test_create_product_with_swatch_attribute_new_attribute_value(
     assert swatch_attribute.values.count() == values_count + 1
 
 
-def test_create_product_with_swatch_attribute_existing_value_using_values_field(
-    staff_api_client,
-    product_type,
-    category,
-    swatch_attribute,
-    permission_manage_products,
-):
-    query = CREATE_PRODUCT_MUTATION
-
```

Comment:
Why are we deleting a test case?

Reply:
Because it's duplicated.

Reply:
Duplicate of?

Reply:
This test is duplicated in this file.

Reply:
Which test case name?

Reply:
The one we're discussing - `test_create_product_with_swatch_attribute_existing_value_using_values_field`